File: 2015/02/2.rb
feet = 0

File.open('input.txt').each do |line|
  l, w, h = line.split('x').map(&:to_i)
  feet += l*w*h + [l+l+w+w, w+w+h+h, h+h+l+l].min
end

puts feet

File: 2015/02/2.py
feet: int = 0

with open("input.txt") as f:
    for line in f:
        l, w, h = map(int, line.split('x'))

        feet += min(l+l+w+w, w+w+h+h, h+h+l+l)
        feet += l*w*h

print(feet)
